src/Take.Blip.Builder/Models/Condition.cs

```diff
@@ -51,6 +50,27 @@ public void Validate()
             {
                 throw new ValidationException("The entity name should be provided if the comparsion source is entity");
             }
+
+            if ((Comparison == ConditionComparison.Exists || Comparison == ConditionComparison.NotExists) && (Values != null && Values.Length >= 0))
+            {
+                throw new ValidationException("The condition values should not be provided if comparison is Exists or NotExists");
+            }
+
+            if ( (Comparison == ConditionComparison.Equals 
```

Comment:
Put operators in the end of each line:
`
 Comparison == ConditionComparison.NotEquals ||
 Comparison == ConditionComparison.Contains ||

(...)
`